### ISSUE TITLE
Update command-line interface incl. usage info

### ISSUE THREAD

**Issue by arnikz** [PBR/QTM#16]

**Comment by arnikz** [comment]
`-help` option works but not `-h`

See below:
 `java -jar target/QTM-1.0-SNAPSHOT.jar -help`

```
HELP pages for QTL Table Miner ++

DESCRIPTION
QTL TableMiner++ is a command-line tool that can retrieve and semantically annotate results of QTL mapping experiments commonly buried in (heterogenous) tables.
java -jar QTM.jar -pmc PMC4266912
ARGUMENTS
    -pmc A list of all pmcids required to be processed. Use comma(,) as a seperator between to ids. For example PMC4266912, PMC2267253
    -o Filename of the output database. This database is in sqlite format.
    -help HELP pages for QTL Table Miner ++
```

rename `-pmc` to  shorter `-i`

 `java -jar target/QTM-1.0-SNAPSHOT.jar -h`

```
========================================================================
QTLTableMiner++ semantic mininig of QTL Tables from scientific articles
========================================================================
____________________________________________________________________________________________________________________________



Processing Article: -h
-----------------------------------------
[Fatal Error] :-1:-1: Premature end of file.
org.xml.sax.SAXParseException; Premature end of file.
at org.apache.xerces.parsers.DOMParser.parse(Unknown Source)
at org.apache.xerces.jaxp.DocumentBuilderImpl.parse(Unknown Source)
at readers.PmcMetaReader.read(PmcMetaReader.java:131)
at qtm.QtmMain.main(QtmMain.java:84)
Problem in reading xml file
Exception in thread "main" java.lang.NullPointerException
at qtm.Article.setNumQTLtables(Article.java:74)
at readers.PmcMetaReader.read(PmcMetaReader.java:167)
at qtm.QtmMain.main(QtmMain.java:84)
```

**Comment by gurnoor1990** [comment]
Still required -i | --input

**Comment by arnikz** [comment]
Implemented the following CLI (2a03ec5):

```
QTM [-v|-h]
QTM [-o FILE_PREFIX] FILE

FILE                                List of full-text articles from Europe PMC. Enter one PMCID per line.
-o|--output FILE_PREFIX             Output files in SQLite/CSV formats. (default: qtl.{db,csv})
-v|--version                        Print software version.
-h|--help                           Print this help message.
```
